server/internal/tasks/index.ts

```diff
@@ -187,6 +252,24 @@ class TaskHandler {
 
     return true;
   }
+
+  /**]
+   * Runs all daily tasks that are scheduled to run once a day.
+   */
+  triggerDailyTasks() {
```

Comment:
Maybe some sort of cron-based scheduler? I think Nitro has one built-in, so maybe we could hook that.

Reply:
Yep, it gets called by by Nitro already